certbot/ocsp.py

```diff
@@ -137,11 +138,16 @@ def _check_ocsp_cryptography(cert_path, chain_path, url):
     builder = builder.add_certificate(cert, issuer, hashes.SHA1())
     request = builder.build()
     request_binary = request.public_bytes(serialization.Encoding.DER)
-    response = requests.post(url, data=request_binary,
-                             headers={'Content-Type': 'application/ocsp-request'})
-    if response.status_code != 200:
+    try:
+        response = requests.post(url, data=request_binary,
+                                 headers={'Content-Type': 'application/ocsp-request'})
+    except requests.exceptions.RequestException:
         logger.info("OCSP check failed for %s (are we offline?)", cert_path)
```

Comment:
nit: Can we use `logger.debug(..., exc_info=True)` to log the exception that occurred here?

Reply:
Done.